CMakeLists.txt

```diff
@@ -388,6 +388,12 @@ install(
   PATTERN "CMakeLists.txt" EXCLUDE
 )
 
+# Create and link locale dir. We always need one even if it's empty, because Widelands is checking for it to prevent crashes.
+add_custom_target(link_locale_dir ALL
+    COMMAND ${CMAKE_COMMAND} -E make_directory locale
```

Comment:
@gunchleoc : Is this really necessary? "create_symlink" does only work on windows with administrative privileges.
And: Even if there is a symlink to the locale data, you'll still have to start the binary explicitely with "--datadir".

I think it would be better to add a "--localedir" parameter or just describe in the build documentation that (if the dev wants to run with locales) to copy the files or make the symlink manually.

The current solution with the symlink is not platform independant.